### ISSUE TITLE
Instant type and Set matcher in PactLambdaDsl

### ISSUE THREAD

**Issue by Sandhyagabbi** [DiUS/pact-jvm#802]
Hi,

I don't see a matcher for Instant type and set. 
Do we already have one or are there any plans to implement it?

**Comment by uglyog** [comment]
Can you explain what you mean by "Instant type and set"?

**Comment by uglyog** [comment]
I've added support for passing example Instants to the timestamp methods in the DSL.

As for supporting Sets, I'm not sure how you envision that to work. The DSL is built on JSON as a model, and JSON only has support for arrays. Where you expecting to do something like

```java
object.set("fooSet", (set) -> set.object((o) -> o.stringValue("bar", "Bar")))
```

which will have no different from the array method, as the underlying collection will be an array.

**Comment by Sandhyagabbi** [comment]
Thank you for the update Ronald. For sets, I was expecting something as you have mentioned above. 
But, using array method resolved our requirement.

**Comment by uglyog** [comment]
This has been released with version 3.6.2
